processor/servicegraphprocessor/processor.go

```diff
@@ -209,6 +211,7 @@ func (p *serviceGraphProcessor) aggregateMetrics(ctx context.Context, td ptrace.
 						e.ClientLatencySec = float64(span.EndTimestamp()-span.StartTimestamp()) / float64(time.Millisecond.Nanoseconds())
 						e.Failed = e.Failed || span.Status().Code() == ptrace.StatusCodeError
 						p.upsertDimensions(clientKind, e.Dimensions, rAttributes, span.Attributes())
+						p.upsertPeerAttributes(NeedToFindAttributes, e.Peer, span.Attributes())
```

Comment:
Only call when `VirtualNodeFeatureEnabled = true`?

Reply:
yes, I forgot this.